tutorials/observability/python/app.py

```diff
@@ -37,4 +35,4 @@ def multiply():
     print(f"Calculating {operand_one} * {operand_two}", flush=True)
     return jsonify(math.ceil(operand_one * operand_two * 100000)/100000)
 
-app.run(port)
+app.run(host="0.0.0.0",port=port)
```

Comment:
Just checking we need to set host?

Reply:
@paulyuk - It is not mandatory. This makes the value being used explicit.

Reply:
@akhilac1 ok, I just want to make sure that still works in a few environments that might have benefited from defaults.  E.g. have you tried this on Windows, WSL, and Mac?  have you tried with Ipv6? 

The other way to ask is are we fixing a problem with this change?  if not we might have lower risk of regressions by not making the change, since this has worked this way for a few years.

Reply:
That said, I'll take it and let's monitor